lib/src/component/prop_mixins.dart

```diff
@@ -43,8 +44,31 @@ abstract class $ReactPropsMixin {
 abstract class _$ReactPropsMixin {
   Map get props;
 
+  // This private field is namespaced to avoid colliding with other classes.
+  @Accessor(key: 'children')
+  dynamic _raw$ReactProps$children;
+
   /// The children that were passed in to this component when it was built.
-  List children;
+  List<dynamic> get children {
```

Comment:
I inlined this implementation into the getter, as opposed to making it a utility function, so that I could do `props.containsKey('children')` below